controllers/admin.go

```diff
@@ -0,0 +1,13 @@
+package controllers
+
+import (
+	"net/http"
+
+	"github.com/gabek/owncast/core/rtmp"
+)
+
+// DisconnectInboundConnection will force-disconnect an inbound stream
+func DisconnectInboundConnection(w http.ResponseWriter, r *http.Request) {
+	rtmp.Disconnect()
+	w.WriteHeader(http.StatusOK)
```

Comment:
Wonder if we should go ahead and write a body back, such as `{ success: true }` so that clients at least get a response back and specifically for the future web client. I'm just thinking we would do something like

```js
const result = await fetch();
await result.json();
```

Or something to that 🤔

Reply:
Yeah there's no reason not to return some standardized success.  I'll add that.